op-dispute-mon/mon/extract/bond_enricher_test.go

```diff
@@ -69,16 +69,18 @@ func TestBondEnricher(t *testing.T) {
 			// Claim 2 CounteredBy is unset
 		}
 		enricher := NewBondEnricher()
-		credits := []*big.Int{big.NewInt(10), big.NewInt(20), big.NewInt(30)}
-		caller := &mockGameCaller{credits: credits}
+		expectedCredits := map[common.Address]*big.Int{
+			expectedRecipients[0]: big.NewInt(10),
+			expectedRecipients[1]: big.NewInt(20),
+			expectedRecipients[2]: big.NewInt(30),
+		}
+		caller := &mockGameCaller{credits: expectedCredits}
 		err := enricher.Enrich(context.Background(), rpcblock.Latest, caller, game)
 		require.NoError(t, err)
 
-		require.Equal(t, expectedRecipients, caller.requestedCredits)
-		expectedCredits := map[common.Address]*big.Int{
-			expectedRecipients[0]: credits[0],
-			expectedRecipients[1]: credits[1],
-			expectedRecipients[2]: credits[2],
+		require.Equal(t, len(expectedRecipients), len(caller.requestedCredits))
+		for _, recipient := range expectedRecipients {
```

Comment:
Tragically `require` does not have a `mapsContentsSame` - even it's `ElementsMatch` doesn't take maps.

Reply:
Yeah I was hoping there'd be a `ContainsInAnyOrder` but no such luck.  `require` needs to go learn some tricks from `AssertJ`.